wsmaster/che-core-api-devfile/src/main/resources/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# This is a Dockerfile allowing to build devfile documentaion by using a docker container.
+# Build step: $ docker build -t eclipse-che-devfile-docs
+# It builds an archive file that can be used by doing later
+#  $  docker run --rm eclipse-che-devfile-docs | tar -C target/docs/ -xf -
+FROM node:8.10.0
+
+RUN apt-get update && \
+    apt-get install -y git \
+    && apt-get -y clean \
+    && rm -rf /var/lib/apt/lists/*
+RUN git clone https://github.com/adobe/jsonschema2md.git
+RUN cd jsonschema2md && npm install && npm link
```

Comment:
Why should we build a master branch of `jsonschema2md`? Won't it be more safety to depends on a tag?
Then we would be able to download sources of tag as archive and do not clone whole repository (not sure that it is an issue, but archive should be smaller)

Reply:
Note that now jsonschema2md repository contains FAILED ci status
![screenshot_20190211_145331](https://user-images.githubusercontent.com/5887312/52564429-d9331680-2e0c-11e9-94ec-024e17f6e68f.png)

Reply:
Also you can avoid installing cli tools for `jsonschema2md` since you don't need it because you can use `node cli.js -d examples/schemas -o examples/docs` instead of `jsonschema2md -d examples/schemas -o examples/docs`

Reply:
it is just simpler to undrerstand

Reply:
master have more bugs fixed & sonme are pending

Reply:
i will fix their build when i will have free time :)

Reply:
yeh. sounds like the important thing to think about.

Reply:
I'm worried about reproducibility. What if we check out some specific version. @sleshchenko @mshaposhnik wdyt?

Reply:
@skabashnyuk I wonder why do we need to clone the whole repository.
But I'm OK with both of checking out tag version or downloading sources by tag as an archive.

Reply:
If we don't want to track tags manually we need to clone whole repo anyway (to find latest tag using ).
Or we can just manually change the tag  when need @skabashnyuk @sleshchenko WDYT

Reply:
> manually change the tag when necessary

+1

Reply:
ok then i will clone using:
`git clone -b '1.1.0' --single-branch git@github.com:adobe/jsonschema2md.git`
that clones only those tag refs

Reply:
It still will contain redundant information about commits, like commit dates, authors, full branch history, like removed lines/files.
But I'm ok with using git if you think that downloading source archive is too difficult to implement and support.